checks/evaluation/dependency_update_tool_test.go

```diff
@@ -88,14 +88,16 @@ func TestDependencyUpdateTool(t *testing.T) {
 					Tools: []checker.Tool{
 						{
 							Name: "DependencyUpdateTool",
-							File: &checker.File{
-								Path: "/etc/dependency-update-tool.conf",
-								Snippet: `
+							File: []checker.File{
+								{
+									Path: "/etc/dependency-update-tool.conf",
+									Snippet: `
 										[dependency-update-tool]
 										enabled = true
 										`,
-								Offset: 0,
-								Type:   0,
+									Offset: 0,
```

Comment:
Offset of 0 can be omitted : it's the default value if you don't specify it

Reply:
same as below, let's discuss this tmr